src/stacked_borrows.rs

```diff
@@ -321,15 +298,15 @@ impl<'tcx> Stack {
     /// the `Size` refers to the specific location in the `AllocRange` that we are
     /// currently checking.
     fn check_protector(
-        item: &Item,
+        item: Item,
         provoking_access: Option<(SbTag, AllocRange, Size, AccessKind)>, // just for debug printing and error messages
         global: &GlobalStateInner,
         alloc_history: &mut AllocHistory,
-    ) -> InterpResult<'tcx> {
+    ) -> InterpResult<'static> {
```

Comment:
Recently in rustc all `InterpResult<'static>` were changed to `'tcx`, so please also suck to `'tcx` here.